mantle/platform/metal.go

```diff
@@ -623,7 +626,34 @@ func (inst *Install) InstallViaISOEmbed(kargs []string, liveIgnition, targetIgni
 			http.Serve(listener, mux)
 		}()
 		baseurl := fmt.Sprintf("http://%s:%d", defaultQemuHostIPv4, port)
-		srcOpt = fmt.Sprintf("--image-url %s/%s", baseurl, metalname)
+
+		// This is subtle but: for the minimal case, while we need networking to fetch the
+		// rootfs, the primary install flow will still rely on osmet. So let's keep srcOpt
+		// as "" to exercise that path. In the future, this could be a separate scenario
+		// (likely we should drop the "offline" naming and have a "remote" tag on the
+		// opposite scenarios instead which fetch the metal image, so then we'd have
+		// "[min]iso-install" and "[min]iso-remote-install").
+		if !minimal {
+			srcOpt = fmt.Sprintf("--image-url %s/%s", baseurl, metalname)
+		}
+
+		if minimal {
+			minisopath := filepath.Join(tempdir, "minimal.iso")
+			// This is obviously also available in the build dir, but to be realistic,
+			// let's take it from --rootfs-output
+			rootfs_path := filepath.Join(tempdir, "rootfs.img")
+			// Ideally we'd use the coreos-installer of the target build here, because it's part
+			// of the test workflow, but that's complex... Sadly, probably easiest is to spin up
+			// a VM just to get the minimal ISO.
+			cmd := exec.Command("coreos-installer", "iso", "extract", "minimal", srcisopath,
```

Comment:
Bikeshed: we already have `iso extract pxe`; is it clear that `iso extract minimal` means a minimal ISO?  Does `iso extract minimal-iso` feel better or worse?

Reply:
I initially did choose `minimal-iso`, but the "iso" repetition bothered me a bit, but agreed using just an adjective is awkward. Good with `minimal-iso` if we can't think of anything nicer (I can't).